.travis.yml

```diff
@@ -32,10 +33,16 @@ addons:
   apt:
     packages:
       - swig
+  firefox: latest
```

Comment:
Needs upstream patching at travis so not a blocker, let's file issues there maybe.